test/integration/scheduler_perf/scheduler_perf_test.go

```diff
@@ -600,8 +600,17 @@ func runWorkload(b *testing.B, tc *testCase, w *workload) []DataItem {
 
 	var mu sync.Mutex
 	var dataItems []DataItem
-	numPodsScheduledPerNamespace := make(map[string]int)
 	nextNodeIndex := 0
+	// numPodsScheduledPerNamespace has all namespaces created in workload and the number of pods they (will) have.
+	// All namespaces listed in numPodsScheduledPerNamespace will be cleaned up.
+	numPodsScheduledPerNamespace := make(map[string]int)
```

Comment:
if the int is not necessary, you can use `sets.String`

Reply:
The number of Pods is used in [`barrierOp`](https://github.com/kubernetes/kubernetes/blob/7588a82bdb1f761ef2a591d18478b0145915af5d/test/integration/scheduler_perf/scheduler_perf_test.go#L488), so `int` is needed.
https://github.com/kubernetes/kubernetes/blob/7588a82bdb1f761ef2a591d18478b0145915af5d/test/integration/scheduler_perf/scheduler_perf_test.go#L804